### DISCUSSION TITLE
This seems to 404 on me for the url for successful builds and deploys...

### DISCUSSION THREAD

**Discussion by k-walsh-gmg** [rossjrw/pr-preview-action#87] [Q&A] [answered]
I am able to get this to succesfully build and deploy but the url only shows a 404 error... I wish I had a bit more for you but nothing really to be seen(works locally).

**Comment by rossjrw** [comment, marked as answer]
_Moved this from Issues to Discussion as it's troubleshooting for a specific setup rather than a bug report or feature request. (If a bug report arises from this discussion feel free to create an issue for it.)_

@k-walsh-gmg That's not enough information to start helping you solve the problem. Do you have links to the repository, pull request, workflow definition or workflow run logs? If it's a private repository or you don't want to link to it for some reason, can you share copies of those? E.g. see https://github.com/rossjrw/pr-preview-action/discussions/77#discussioncomment-8249382 for an example
